audb/core/info.py

```diff
@@ -13,6 +13,37 @@
 )
 
 
+def attachments(
+        name: str,
+        *,
+        version: str = None,
+        cache_root: str = None,
+) -> typing.Dict:
```

Comment:
I think we could be more precise here:

```suggestion
) -> typing.Dict[str, audformat.Attachment]:
```

Though we should probably change it also for other functions in this file. So maybe move it to a separate PR?

Reply:
I created https://github.com/audeering/audb/issues/281